### PR TITLE
Feature/k8s

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

**Review by kasperg — COMMENTED** [review]
Remember to remove parts of the old deployment setup.

This includes:

- [Github workflows for deployment](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches/blob/feature/k8s/.github/workflows/push.yml#L92-L120)
- [Parts](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches/blob/master/Envoy.blade.php) [of](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches/blob/master/composer.json#L35) the old deployment setup using Envoy.
- [Documentation referring to the old deployment setup](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches#server-setup)

**Review by kasperg — CHANGES_REQUESTED** [review]
My comments regarding danskernesdigitalebibliotek/ddb-material-list#1 also apply here.

Also see my previous comment about removing [parts](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches/blob/master/Envoy.blade.php) [of](https://github.com/danskernesdigitalebibliotek/ddb-follow-searches/blob/master/composer.json#L35) the old deployment setup using Envoy.

**Comment by kasperg** [comment]
The specification test failure is not related to this PR. It will be fixed when #4 is merged.

**Comment by cableman** [comment]
Updated the code with changes from this review and the one for material list.

I regard to en Envoy it seams to be an integrated part of the Larvel and as I don't known anything about Larvel I think you should make another PR removing it. It do not seam to block anything here.

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -44,3 +46,14 @@ jobs:
...
         REVIEWDOG_TOKEN: ${{ secrets.REVIEWDOG_TOKEN }}
+
+  helm_validation:
+    name: Helm chart validation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: azure/setup-helm@v1
+        name: Install helm
+        with:
+          version: v3.3.4
```
Comment (kasperg) [inline .github/workflows/pull_request.yml:57]: Same quibbles are for material list regarding versioning.

**Review by kasperg — COMMENTED** [review]
> I regard to en Envoy it seams to be an integrated part of the Larvel and as I don't known anything about Larvel I think you should make another PR removing it. It do not seam to block anything here.

It is a Composer package and a configuation file that needs to be removed. I linked to the parts in my comment. I have removed them it a commit added to the PR in a25939d.

**Pull request merged**